cypress.json

```diff
@@ -20,6 +20,7 @@
     "VISBUILDER_ENABLED": true,
     "DATASOURCE_MANAGEMENT_ENABLED": false,
     "ML_COMMONS_DASHBOARDS_ENABLED": true,
-    "WAIT_FOR_LOADER_BUFFER_MS": 0
+    "WAIT_FOR_LOADER_BUFFER_MS": 0,
+    "NO_COMMAND_LOG": 1
```

Comment:
Q: could you elaborate this change? seems not related to this PR

Reply:
On Chrome 117 the test will hang because of browser crash. According to the [doc](https://docs.cypress.io/guides/references/troubleshooting#Disable-the-Command-Log) here it seems the only option we can choose.

It is better to leave a comment here but it seems we can not write comment in json file.